.github/workflows/test_and_release.yml

```diff
@@ -0,0 +1,127 @@
+name: tests
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+  pull_request:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+-dev
+
+jobs:
+  test:
+    name: Run SDK tests
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Enable NPM cache
+        uses: actions/cache@v2
+        with:
+          path: '~/.npm'
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Cache Docker layers
+        uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true
+        with:
+          concurrency: 8
+
+      - name: Check NPM package lock version is updated
+        run: npm run check-package # TODO: does this exist as an action already?
+
+      - name: Install NPM dependencies
+        run: npm ci
+
+      - name: Create dotenv file
+        run: cp .env.example .env
+
+      - uses: dashevo/gh-action-runtime-vars@master
+        id: runtime-vars
+        if: ${{ contains(github.event.pull_request.body, '/set-var')}}
+        with:
+          pull-request-body: ${{ github.event.pull_request.body }}
```

Comment:
Why not just read it directly in action?

Reply:
I opened a PR suggesting to do it this way 2 weeks ago: https://github.com/dashevo/gh-action-pr-vars/pull/1

Reply:
Merged! 👍